README.md

```diff
@@ -79,42 +79,33 @@ The above will provide you with a working build of the application that you can
 run.  Below you will find tasks which are someone outside of the normal
 development process, but are documented for future reference.
 
-### Setting up the App Icon
-Using the above instructions, the app will build using the default Cordova app
-icon. To change the app icon, edit `cordova/config.xml`.
+### Configuring the Splash Screen and App Icon
 
-In that file, add the following line in the top level of the xml:
+Add the following to the top level of the XML in cordova/config.xml:
 
 ```
-<icon src="../resources/icons/Icon@2x.png" />
+    <icon src="../resources/branding/drawable-ldpi/icon.png" density="ldpi" width="36" height="36" />
+    <icon src="../resources/branding/drawable-mdpi/icon.png" density="mdpi" width="48" height="48" />
+    <icon src="../resources/branding/drawable-hdpi/icon.png" density="hdpi" width="72" height="72" />
+    <icon src="../resources/branding/drawable-xhdpi/icon.png" density="xhdpi" width="96" height="96" />
+    <icon src="../resources/branding/drawable/icon.png" />
+
+    <preference name="AutoHideSplashScreen" value="false" />
+
+    <splash src="../resources/branding/drawapble-ldpi/splash.png" density="port-ldpi" width="480" height="800" />
```

Comment:
@CShapleigh @bbesmanoff you two are bad reviewers, you missed the typo. I'll throw in another commit to fix it before I merge.